src/Umbraco.Web.UI.Client/src/common/resources/redirecturls.resource.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * @ngdoc service
+ * @name umbraco.resources.redirectUrlResource
+ * @function
+ *
+ * @description
+ * Used by the redirect url dashboard to get urls and send requests to remove redirects.
+ */
+(function() {
+    'use strict';
+
+    function redirectUrlsResource($http, umbRequestHelper) {
+
+        var redirectBaseUrl = "backoffice/api/RedirectUrlManagement/";
```

Comment:
I am aware of that.. This was how far I could get it. Now I need a bit of assistance to get the last part fixed.